### ISSUE TITLE
fortigate issue - "Execution expired"

### ISSUE THREAD

**Issue by malkovitch** [ytti/oxidized#150]
Hello,

I'v got a problem with a fortigate 40c.

Here are the log :

I, [2015-07-21T08:56:26.799716 #29194]  INFO -- : Oxidized starting, running as pid 29194
I, [2015-07-21T08:56:26.869345 #29194]  INFO -- : Loaded 5 nodes
W, [2015-07-21T08:57:42.269092 #29194]  WARN -- : 192.168.253.183 raised Timeout::Error with msg "execution expired"
W, [2015-07-21T08:57:42.820748 #29194]  WARN -- : 192.168.253.183 status no_connection, retry attempt 1
W, [2015-07-21T08:58:30.279628 #29194]  WARN -- : 192.168.253.183 raised Timeout::Error with msg "execution expired"
W, [2015-07-21T08:58:30.830949 #29194]  WARN -- : 192.168.253.183 status no_connection, retry attempt 2
W, [2015-07-21T08:59:18.276856 #29194]  WARN -- : 192.168.253.183 raised Timeout::Error with msg "execution expired"
W, [2015-07-21T08:59:18.839865 #29194]  WARN -- : 192.168.253.183 status no_connection, retry attempt 3
W, [2015-07-21T09:00:06.306186 #29194]  WARN -- : 192.168.253.183 raised Timeout::Error with msg "execution expired"
W, [2015-07-21T09:00:06.848584 #29194]  WARN -- : 192.168.253.183 status no_connection, retries exhausted, giving up

And here the end of the input ssh debug log :

FGT40C-Test # show

config system global
    set fgd-alert-subscription advisory latest-threat
    set hostname "FGT40C-Test"
    set timezone 04
end
config system accprofile
    edit "prof_admin"
        set admingrp read-write
        set authgrp read-write
        set endpoint-control-grp read-write
        set fwgrp read-write
        set loggrp read-write
        unset menu-file
        set mntgrp read-write
        set netgrp read-write
        set routegrp read-write
        set sysgrp read-write
        set updategrp read-write
        set utmgrp read-write
        set vpngrp read-write
        set wanoptgrp read-write
        set wifi read-write
--More--

It looks like the "show" command is not executed completely, stuck on --More--

I hope you can help me,

Thank you.

**Comment by ytti** [comment]
Is there way to stop fortigate from paging? Perhaps @aakso can help with this.

**Comment by malkovitch** [comment]
Yes,

On a fortigate you have to do :

config system console
set output standard

Now it works !

But the thing is i don't wa

**Comment by itscassa** [comment]
@malkovitch You can do this using the pre_logout feature

**Comment by aakso** [comment]
Please make a pull request if you patch this.

If I remember correctly FortiOS will persist the pager configuration so maybe we need to detect the current setting and set it back when we are finished.

**Comment by andreas1o** [comment]
Fixed this also but had to do a awkard workaround.. as I see it catch 22.

In a vdom environment we have to do 
config global 
then 
config system console 
set output standard 
end 
end 

Right now we can't get the 
get system status
 to be good since it per default does more. 

So I had to just add some commands that will fail on a none vdom enabled router 
but works on a vdom enabled router. Maybe the trick would be to create a class for fortgatevdom ? 

I'll try to pull this to the repository... and comments are welcome.
